doc/lua_api.md

```diff
@@ -61,7 +61,7 @@ The game directory can contain the following files:
 * `game.conf`, with the following keys:
     * `title`: Required, a human-readable title to address the game, e.g. `title = Minetest Game`.
     * `name`: (Deprecated) same as title.
-    * `description`: Short description to be shown in the content tab
+    * `description`: Short description to be shown in the content tab. See [Translating content meta].
```

Comment:
Is this supposed to be a short format of link? If so: it does not render as a link in the web view.

Reply:
fixed